### PR TITLE
Remove default grids (content)

### PR DESCRIPTION
**Pull request by Zumorica**
Requires https://github.com/space-wizards/RobustToolbox/pull/1322

### CODE REVIEW

File: Content.Client/State/GameScreenBase.cs [hunk 1]
```diff
@@ -209,11 +209,12 @@ private void OnKeyBindStateChanged(BoundKeyEventArgs args)
             var mousePosWorld = EyeManager.ScreenToMap(args.PointerLocation);
             var entityToClick = GetEntityUnderPosition(mousePosWorld);
 
-            if (!MapManager.TryFindGridAt(mousePosWorld, out var grid))
-                grid = MapManager.GetDefaultGrid(mousePosWorld.MapId);
+            IMapGrid grid;
+            if (!MapManager.TryFindGridAt(mousePosWorld, out grid))
+                grid = null;
```
Comment (RemieRichards) [inline Content.Client/State/GameScreenBase.cs:214]: Jesus this is annoying to look at.

Can't you just move the getting of the `FullInputCmdMessage` arg out so we don't need this?
or at least invert the logic.

```cs
IMapGrid grid = null;
MapManager.TryFindGridAt(mousePosWorld, out grid);
```

File: Content.Client/UserInterface/ItemSlotManager.cs [hunk 2]
```diff
@@ -70,18 +70,20 @@ public bool OnButtonPressed(GUIBoundKeyEventArgs args, IEntity item)
                 var func = args.Function;
                 var funcId = _inputManager.NetworkBindMap.KeyFunctionID(args.Function);
 
+
                 var mousePosWorld = _eyeManager.ScreenToMap(args.PointerLocation);
+
                 if (!_mapManager.TryFindGridAt(mousePosWorld, out var grid))
-                    grid = _mapManager.GetDefaultGrid(mousePosWorld.MapId);
+                    grid = null;
```
Comment (RemieRichards) [inline Content.Client/UserInterface/ItemSlotManager.cs:77]: Ditto as above.

File: Content.Client/State/GameScreenBase.cs (hunk 1 above)
Comment (Zumorica) [reply Content.Client/State/GameScreenBase.cs]: Changed.

**Comment by Zumorica** [comment]
Ready for review.

**Review by PJB3005 — APPROVED** [review]

**Pull request merged**